saleor/graphql/product/tests/test_attributes.py

```diff
@@ -419,7 +419,8 @@ def test_assign_variant_attribute_having_multiselect_input_type(
 ):
     """The assignAttribute mutation should raise an error when trying
     to use an attribute as a variant attribute when
-    the attribute's input type doesn't support variants"""
+    the attribute's input type doesn't support variants
+    TODO: except is not"""
```

Comment:
Is it a good place for `TODO`? Or maybe it is something left by mistake?

Reply:
It's just a hook to get an attention for that test :) as we spoke, I remove entire docstring straight away.